configure.ac

```diff
@@ -209,11 +209,7 @@ PKG_CHECK_MODULES([SSSNSSIDMAP], [sss_nss_idmap >= 1.13.90])
 dnl ---------------------------------------------------------------------------
 dnl - Check for POPT
 dnl ---------------------------------------------------------------------------
-
-POPT_LIBS=
-AC_CHECK_HEADER(popt.h, [], [AC_MSG_ERROR([popt.h not found])])
-AC_CHECK_LIB(popt, poptGetContext, [POPT_LIBS="-lpopt"])
-AC_SUBST(POPT_LIBS)
+PKG_CHECK_MODULES([POPT], [popt])
```

Comment:
It prints nice informational message how to work it around and it actually works on RHEL 7, so I'm not going to comlicate the build system for crappy packages.